solutions/swb-reference/integration-tests/tests/isolated/environments/start.test.ts

```diff
@@ -53,4 +60,69 @@ describe('environment start negative tests', () => {
       );
     }
   });
+
+  test('project was deleted', async () => {
+    const dataSetName = randomTextGenerator.getFakeText('integration-test-dataSet');
+
+    const { data: costCenter } = await adminSession.resources.costCenters.create({
+      name: `${dataSetName} cost center`,
+      accountId: setup.getSettings().get('defaultHostingAccountId'),
+      description: 'a test object'
+    });
+
+    const { data: createdProject } = await adminSession.resources.projects.create({
+      name: `${dataSetName} project`,
+      description: 'test description',
+      costCenterId: costCenter.id
+    });
+
+    const projectId = createdProject.id;
+
+    const dataSetBody = CreateDataSetRequestParser.parse({
+      storageName: settings.get('DataSetsBucketName'),
+      awsAccountId: settings.get('mainAccountId'),
+      path: dataSetName, // using same name to help potential troubleshooting
+      name: dataSetName,
+      region: settings.get('awsRegion'),
+      owner: getProjectAdminRole(createdProject.id),
+      ownerType: 'GROUP',
+      type: 'internal',
+      permissions: [
+        {
```

Comment:
do you need this permission for this test?

Reply:
TBH I'm not sure, I've copied this test from other location and kept it intact to make sure it's working properly. Since the comments are addressing tests Is it ok to resolve the comments after code freeze as low risk changes?